types/order.d.ts

```diff
@@ -0,0 +1,89 @@
+export interface Options {
+    readonly fields?: Array<string>,
+    readonly eager?: Boolean,
+    readonly eagerL?: Boolean,
+    readonly map?: Boolean,
+    readonly rules?: Boolean,
+    readonly meta?: Boolean,
+    readonly build?: Boolean,
+    readonly fill?: Boolean,
+    readonly resolveA?: Boolean,
+    readonly skip?: Number,
+    readonly limit?: Number,
+    readonly sort?: Array<Object>,
+    readonly raiseE?: Boolean,
+}
+
+export enum Currency {

```

Comment:
Since we're keeping currency as an open set in ripe-twitch let's do the same here (`string`)